ValenceClientSide/Program.cs

```diff
@@ -19,6 +19,10 @@ class WhoAmIResponse {
     class Program
     {
         private static ID2LUserContext  InterceptUserTokens( HostSpec host, ID2LAppContext appContext ) {
+            
+            // TLS 1.1 and below has been deprecated. This ensures we use TLS 1.2
+            ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12;
```

Comment:
https://docs.microsoft.com/en-us/dotnet/api/system.net.securityprotocoltype?view=net-5.0
SystemDefault is actually what we want (and it's the default)
It will defer to the OS.
We should update this sample to be .NET 5+